### PR TITLE
Update info about HTTP API

### PR DESCRIPTION
**Pull request by G1gg1L3s**
Because of https://github.com/cossacklabs/acra/pull/550 we have a new cli flag and a couple of limitations less. This PR describes it.

I've checked, but it may not be the all places that should be updated. If you remember something else, please let me know.

Merge this PR after the https://github.com/cossacklabs/acra/pull/550.

### CODE REVIEW

File: content/acra/configuring-maintaining/general-configuration/acra-server/_index.md
Comment (vixentael) [inline content/acra/configuring-maintaining/general-configuration/acra-server/_index.md:260]: then why default is false? :)

Comment (G1gg1L3s) [reply content/acra/configuring-maintaining/general-configuration/acra-server/_index.md:260]: Backward compatibility ☹️
I guess, in the future we will make it `true` by default. However it's not that straightforward, because the clients should specify TLS keys and certificates, which they don't always do (like in situations when they play with the database and acra locally, so there is no need of specifying the TLS params).

Comment (vixentael) [reply content/acra/configuring-maintaining/general-configuration/acra-server/_index.md:260]: okay.jpg

**Review by vixentael — APPROVED** [review]

**Comment by vixentael** [comment]
@Lagovas will you take a look, or should we merge?

File: content/acra/configuring-maintaining/general-configuration/acra-server/_index.md
Comment (Lagovas) [reply content/acra/configuring-maintaining/general-configuration/acra-server/_index.md:260]: hm. looked one more time and see http_api_use_tls and http_api_enable... and I think that we should rename it to something like http_api_tls_transport_enable... I suggested it @G1gg1L3s in Slack and will repeat it in #550. So lets merge this PR after merge into acra repo.

**Review by vixentael — APPROVED** [review]

**Review by Lagovas — APPROVED** [review]

**Pull request merged**
